domains/0v90.json

```diff
@@ -1,14 +1,16 @@
 {
+    "description": "Personal Portfolio Website",
+    "repo": "https://github.com/0v90/0v90.github.io",
     "owner": {
         "username": "0v90",
+        "discord": 651519394673065989,
         "email": "",
-        "Discord": "0v90"
+        
     },
     "record": {
-        "A": [
-            "35.200.171.190",
-            "35.240.84.96",
-            "34.132.146.171"
-        ]
+        "CNAME": "0v90.github.io",
+        "TXT": "_github-pages-challenge-0v90",
```

Comment:
You cannot use CNAME with TXT (Follow the docs here on how to do the txt verification https://www.is-a.dev/docs/github-pages-verification/)